From e80c4836299f74eda40db99e350ac4c3b8f7d16a Mon Sep 17 00:00:00 2001
From: Stefan Assmann <sassmann@redhat.com>
Date: Tue, 2 Sep 2014 13:11:34 -0400
Subject: [ethernet] i40e/i40evf: add driver version string to driver version command

Message-id: <1409663591-32531-52-git-send-email-sassmann@redhat.com>
Patchwork-id: 90532
O-Subject: [RHEL7.1 PATCH 051/148] i40e/i40evf: add driver version string to driver version command
Bugzilla: 1091127
RH-Acked-by: Prarit Bhargava <prarit@redhat.com>
RH-Acked-by: Tony Camuso <tcamuso@redhat.com>

From: Shannon Nelson <shannon.nelson@intel.com>

The driver version string was added to this struct to be passed
down through the firmware to low-level NC-SI functions. We tell
the firmware about the length of the ASCII string not counting
any terminating null.

Change-ID: I09ac98ff9b869e8661c55fc6a5c98808fc280c91
Signed-off-by: Shannon Nelson <shannon.nelson@intel.com>
Acked-by: Anjali Singhai <anjali.singhai@intel.com>
Tested-by: Kavindya Deegala <kavindya.s.deegala@intel.com>
Signed-off-by: Jeff Kirsher <jeffrey.t.kirsher@intel.com>
Signed-off-by: Stefan Assmann <sassmann@redhat.com>
(cherry picked from commit d2466013bbf77aee88349b2be258d37a8b9e8bea)
---

Signed-off-by: Jarod Wilson <jarod@redhat.com>

diff --git a/drivers/net/ethernet/intel/i40evf/i40e_type.h b/drivers/net/ethernet/intel/i40evf/i40e_type.h
index 4673b33..449a3be 100644
--- a/drivers/net/ethernet/intel/i40evf/i40e_type.h
+++ b/drivers/net/ethernet/intel/i40evf/i40e_type.h
@@ -415,6 +415,7 @@ struct i40e_driver_version {
  u8 minor_version;
  u8 build_version;
  u8 subbuild_version;
+ u8 driver_string[32];
 };
 
 /* RX Descriptors */
-- 
1.7.1